### ISSUE TITLE
Collapsed Section don't expand on frontend validation error

### ISSUE THREAD

**Issue by ziming** [laravel-filament/filament#2231]
### Browser

Firefox Developer Edition

### Package

filament/forms

### Package Version

2.11.4

### Laravel Version

9.9

### Livewire Version

2.10.5

### PHP Version

8.1.5

### Bug description

When a section is collapsed and I press submit, the submit is blocked but the section did not expand to show the field that fails the frontend validation.

### Steps to reproduce

Sample code to reproduce

```php
class CreateBook extends Component implements HasForms
{
    use InteractsWithForms;

}

protected function getFormSchema(): array
    {
        return [
            Section::make('Basic Loan Application Request')
                ->description('Some Basic Questions about your loan request.')
                ->schema([
                    TextInput::make('title')->required(), // leave it blank before pressing submit later.
                ])->collapsible(); // collapse this section later before pressing submit later.
        ];
    }
```

**Comment by danharrin** [comment]
The logic we use to open collapsed sections when there is a validation error is based on the backend, since that's where validation is handled. However, for some fields like textinput, there is also frontend validation.

This bug occurs because the frontend validation is ran before backend on concealed fields, and then the section is never uncollapsed.

Any clue how to conditionally disable frontend validation for all child elements of a `<div>`, @ryangjchandler?

**Comment by ziming** [comment]
I'm not sure if there r differences between tabs and section but I somewhat recall the nova tabs package handling this issue pretty well with frontend validation. jumping to the right tab

may want to check it out

**Comment by danharrin** [comment]
@RVxLab you maintained the Nova Tabs package for a while, right? Can you give us any hints about how you handled frontend validation errors?

**Comment by RVxLab** [comment]
I indeed maintained Nova Tabs for some time. However, Nova Tabs didn't handle validation at all, it delegated everything to the underlying components and let Nova handle the rest.

**Comment by danharrin** [comment]
@RVxLab so it didn't focus the invalid tab or anything?

**Comment by ziming** [comment]
I come across this issue that appear to address the issue

https://github.com/eminiarts/nova-tabs/issues/21

**Comment by danharrin** [comment]
This has been fixed by #2320. I've disabled frontend validation for components inside tabs or collapsible sections. Technically, we could have had a solution by catching the `invalid` event, but some browsers are still unable to focus the component, even after the section is expanded.
